server/py/services/alerts/tests/unit/db/test_alerts.py

```diff
@@ -15,42 +15,32 @@
 
 from datetime import datetime, timezone
 
-import mlrun.common.schemas
-import mlrun.model
+import mlrun.common.schemas.alert as alert_objects
```

Comment:
```suggestion
import mlrun.common.schemas.alert
```
And then use objects with full link following the MLRun development best practices

Reply:
Using `as` is not against the best practices (`from` is frowned upon):

> When importing from local code, NEVER use: `from X import Y` instead use: `import X` or `import X as Y`